### PR TITLE
feat: implement Unmarshal and bigquery.ValueLoader

### PR DESCRIPTION
**Pull request by odsod**
- docs: update contact details in code of conduct
- feat: implement Unmarshal and bigquery.ValueLoader

### CODE REVIEW

File: encoding/protobq/decode_test.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+			name: "expr.Value (bool)",
+			row: map[string]bigquery.Value{
+				"double_value": float64(42),
+			},
+			expected: &expr.Value{
+				Kind: &expr.Value_DoubleValue{
+					DoubleValue: 42,
+				},
+			},
+		},
```
Comment (Jassob) [inline encoding/protobq/decode_test.go:78]: I'm missing tests that makes use of different unmarshal options.

File: encoding/protobq/decode_test.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+			schema: bigquery.Schema{
+				{Name: "bool_value", Type: bigquery.BooleanFieldType},
+				{Name: "double_value", Type: bigquery.FloatFieldType},
+			},
+			expected: &expr.Value{
+				Kind: &expr.Value_DoubleValue{
+					DoubleValue: 42,
+				},
+			},
+		},
```
Comment (Jassob) [inline encoding/protobq/decode_test.go:184]: Missing tests with non-standard options here as well

**Review by Jassob — COMMENTED** [review]
Looks really solid to me, well done! :star:

I'm only missing tests that test that the options modifies the behavior as expected, other than that I'd say we're golden!

File: encoding/protobq/decode_test.go (hunk 1 above)
Comment (odsod) [reply encoding/protobq/decode_test.go:82]: Done!

File: encoding/protobq/decode_test.go (hunk 2 above)
Comment (odsod) [reply encoding/protobq/decode_test.go:219]: Done!

**Review by Jassob — APPROVED** [review]
Great work, well done! :+1:

**Comment by odsod** [comment]
@Jassob Thanks! :raised_hands:

**Pull request merged**